.github/workflows/tests.yml

```diff
@@ -128,7 +128,7 @@ jobs:
     services:
       postgres:
         # Docker Hub image
-        image: postgres
+        image: postgres:alpine
```

Comment:
To keep it as close as the current behaviour, I didn't add version. Currently will be `postgresql:15.3-alpine3.18`. 

See https://hub.docker.com/_/postgres

Reply:
Actually moved to a full tag